### PR TITLE
Use tf.nn.log_softmax for numerical stability

### PR DESCRIPTION
**Pull request by steven0129**

### CODE REVIEW

**Comment by steven0129** [comment]
Hi @leohsuofnthu 

Maybe you need this PR, or YOLACT will unstable when calculating OHEM loss.
